packages/strapi-design-system/src/Tabs/Tabs.tsx

```diff
@@ -185,7 +207,13 @@ export const Tab = ({ disabled, id, children, variant, hasError, index, selected
     console.warn('The "hasError" prop is only available for the "simple" variant.');
   }
 
-  const showRightBorder = selectedTabIndex - 1 === index;
+  const showRightBorder = () => {
+    if (selectedTabIndex) {
+      return selectedTabIndex - 1 === index;
+    }
+
+    return false;
+  };
```

Comment:
How come you've made this a function?

Reply:
Probably just to make something work then forgot about it. I updated with a better solution... I think.